includes/class-wporg-gp-translation-events-translation-listener.php

```diff
@@ -157,10 +157,11 @@ function ( $event ) use ( $at ) {
 	// phpcs:disable Generic.CodeAnalysis.UnusedFunctionParameter.FoundAfterLastUsed
 	// phpcs:disable Generic.CodeAnalysis.UnusedFunctionParameter.Found
 	private function select_events_user_is_registered_for( array $events, int $user_id ): array {
+		$attending_events = get_user_meta( $user_id, 'translation-events-attending', true );
 		return array_filter(
 			$events,
-			function ( $event ) use ( $attending_event_ids ) {
-				return isset( $attending_event_ids[ $event->ID ] );
+			function ( $event ) use ( $attending_events ) {
```

Comment:
Is the `$events` variable passed in from the outside not what we are looking for?
```suggestion
			function ( $event ) use ( $events ) {
```

Reply:
The `$events` passed from outside is the array of [active events](https://github.com/WordPress/wporg-gp-translation-events/blob/9ae7586b67d83c02a79958d27424861f91ff678d/includes/class-wporg-gp-translation-events-translation-listener.php#L62). We then want to filter this array to ensure we have only the events that the user is attending.

Reply:
Ok, I see. Could we then please use the `USER_META_KEY_ATTENDING` constant like it's used here? https://github.com/WordPress/wporg-gp-translation-events/blob/d21f915b7f3a0237fd5f444615606690057e1f63/includes/class-wporg-gp-translation-events-route.php#L184